dockstore-webservice/src/main/resources/migrations.1.9.0.xml

```diff
@@ -50,10 +50,11 @@
 
     <changeSet author="aduncan (generated)" id="addLegacyVersionColumn">
```

Comment:
FYI, this could lead to some subtle errors. My understanding is that if someone (e.g. a developer or staging) has run this migration already. 
Liquibase will not re-run it with these changes. So they may think they have an up-to-date database but actually something slightly different.

Reply:
I figured we had leeway until an actual release; I personally reload the data and run the migrations all the time, so it wouldn't have affected me. :) I'll move it out